examples/Mediawiki/Patrolling.yaml

```diff
@@ -0,0 +1,130 @@
+# This should probably me merged with the Autopatrolling suite.
+# Separate tests, but same suite.
+suite: Autopatrolling
+description: Testing default autopatrolling rights
+type: RPC
+
+use-fixtures:
+  - Alice
+  - Mindy
+
+prototypes:
+  edit:
+      request:
+        method: post
+        path: api.php
+        parameters:
+          action: edit
+          format: json
+        form-data:
+          token: p/var: /session/edit-token
+      response:
+        body:
+          edit:
+            result: Success
+            revid: p/grab: revid
+  rc:
+      use-session: Mindy # you need patroller rights to see patrol flags
+      request:
+          path: api.php
+          parameters:
+            action: query
+            list: recentchanges
+            rctitle: p/var: title
+            rcprop: patrolled
+            format: json
+
+  # Get a patrol token
+  patrol-token:
+   request:
+      path: api.php
+      parameters:
+        action: query
+        meta: tokens
+        type: patrol
+        format: json
+    response:
+      body:
+        query:
+          tokens:
+            patrol: p/grab: patrol-token
+
+  # Do patrol
+  patrol-token:
```

Comment:
This prototype and the one above has the same name

Reply:
good catch :) should just be called "patrol", of course.